starfish/image/_stack.py

```diff
@@ -115,18 +114,31 @@ def __init__(self, image_partition):
 
     @classmethod
     def from_url(cls, relativeurl, baseurl):
```

Comment:
I still get confused about how relativeurl + baseurl should be used here. Could you add an example (or more than one, if there are multiple common use cases)? I think it would help me not keep rewriting code that relates to ImageStack.

Reply:
Updated.  Please let me know if it addresses your concern.